### ISSUE TITLE
Error to get taxa - BOM

### ISSUE THREAD

**Issue by GustavoHeringer** [fgabriel1891/BiodiversityObservationsMiner#5]
Dear Gabriel Muñoz,

I’m trying to explore the Biodiversity Observations Miner, but when I got the message “Error: An error has occurred.  Check your logs or contact the app author for clarification”. I have been tried using chrome and edge and, also tried to find data by taxonomic name and key-words.

The results were always the same. Could you help me with the use of the app?

I’m very interested to use BOM to get plant records from urban areas datasets. Would be possible to use BOM to get this kind of data? If no, do you have any suggestions?

Thank you very much.

Best regards,
Gustavo
gustavoheringer@hotmail.com

**Comment by fgabriel1891** [comment]
Hi Gustavo, 

I'm sorry to have replied late. Unfortunately, the API connection to NCBI has reached the limits so it's no longer possible to use the application from the webserver. Please run the application locally. You can do it by downloading the zip folder from this gitHub repository. Please let me know if you have other issues.  

Thanks, 
Gabriel

**Comment by fgabriel1891** [comment]
Yes, its possible to use BOM kind of approaches to help you discover any kind of records from literature. do you have literature compiled were you would like to extract information?
